src/libraries/System.Reflection.Metadata/tests/Metadata/Ecma335/Encoding/BlobEncodersTests.cs

```diff
@@ -27,6 +27,21 @@ public void BlobEncoder_FieldSignature()
             Assert.Same(b, s.Builder);
         }
 
+        [Fact]
+        public void BlobEncoder_Field()
+        {
+            var b = new BlobBuilder();
+            var e = new BlobEncoder();
+            Assert.Same(b, e.Builder);
+
+            var f = e.Field();
+            f.CustomModifiers()
+                .AddModifier(MetadataTokens.TypeDefinitionHandle(1), isOptional: true)
+                .AddModifier(MetadataTokens.TypeDefinitionHandle(2), isOptional: false);
+            f.Type(isByRef: true).Object();
+            AssertEx.Equal(new byte[] { 0x06, 0x20, 0x04, 0x1F, 0x08, 0x10, 0x1C }, b.ToArray());
+        }
```

Comment:
> } [](http://example.com/codeflow?start=7&length=2)

Please consider adding a test case calling `f.Type()`, with no explicit `isByRef` parameter. Feel free to add it to this test after calling `b.Clear()`, to match the style of other test methods.